File: library/src/main/java/com/alamkanak/weekview/DateTimeInterpreter.java
package com.alamkanak.weekview;

import android.support.annotation.NonNull;

import java.util.Calendar;

/**
 * Created by Raquib on 1/6/2015.
 */
public interface DateTimeInterpreter {

    String interpretDate(@NonNull Calendar date);
    String interpretTime(int hour);

}

File: core/src/main/java/com/alamkanak/weekview/DateTimeInterpreter.kt
package com.alamkanak.weekview

import java.util.Calendar

@Deprecated("Use setDateFormatter() and setTimeFormatter() instead.")
interface DateTimeInterpreter {
    fun onSetNumberOfDays(days: Int) {
        // Free ad space
    }
    fun interpretDate(date: Calendar): String
    fun interpretTime(hour: Int): String
}
